packages/vue-template-compiler/README.md

```diff
@@ -2,7 +2,7 @@
 
 > This package is auto-generated. For pull requests please see [src/platforms/web/entry-compiler.js](https://github.com/vuejs/vue/tree/dev/src/platforms/web/entry-compiler.js).
 
-This package can be used to pre-compile Vue 2.0 templates into render functions to avoid runtime-compilation overhead and CSP restrictions. You will only need it if you are writing build tools with very specific needs. In most cases you should be using [vue-loader](https://github.com/vuejs/vue-loader) or [vueify](https://github.com/vuejs/vueify) instead, both of which use this package internally.
+This package can be used to pre-compile Vue 2.0 templates into render functions to avoid runtime-compilation overhead and CSP restrictions. You will only need it if you are writing build tools with very specific needs. In most cases you should be using [`vue-loader`](https://github.com/vuejs/vue-loader) or [`vueify`](https://github.com/vuejs/vueify) instead, both of which use this package internally.
```

Comment:
I don't think those should be wrapped with backticks, they're just package names after all

Reply:
To reference the current HEAD version https://raw.githubusercontent.com/vuejs/vue/master/packages/vue-template-compiler/README.md the string `vue-loader` shows up 4 times.  In 3 of those cases the name is wrapped in backticks:

```markdown
In most cases you should be using [vue-loader](https://github.com/vuejs/vue-loader) ...
... e.g `vue-loader` and `vueify`.**
This is used by default in `vue-loader@>=12` ...
This is used in SFC build tools like `vue-loader` and `vueify`.
```

Reply:
oh snap